CHANGELOG.md

```diff
@@ -5,7 +5,40 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
```

Comment:
we should get rid of CHANGELOG.md and just use git releases

Reply:
https://github.com/conqa/serverless-openapi-documentation/issues/23